cmake/Corrosion.cmake

```diff
@@ -1739,16 +1746,22 @@ function(corrosion_add_cxxbridge cxx_target)
                     --output "${source_placement_dir}/${cxx_source}"
                     --include "${cxx_target}/${cxx_header}"
             DEPENDS "cxxbridge_v${cxx_required_version}" "${rust_source_path}"
-            COMMENT "Generating cxx bindings for crate ${_arg_CRATE}"
+            COMMENT "Generating cxx bindings for crate ${_arg_CRATE} and file src/${filepath}"
         )
 
-        target_sources(${cxx_target}
-            PRIVATE
-                "${header_placement_dir}/${cxx_header}"
-                "${generated_dir}/include/rust/cxx.h"
-                "${source_placement_dir}/${cxx_source}"
-        )
+        list(APPEND GENERATED_FILES
+            "${header_placement_dir}/${cxx_header}"
+            "${generated_dir}/include/rust/cxx.h"
```

Comment:
I think this line can be removed, since cxx.h is already added in the  set command a couple of lines above.

Reply:
Hm, seems like you're right.
The previous code included it there for some reason, but I think now it shouldn't be necessary anymore :thinking: 
Will fix.

Reply:
Done